sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGenerateExec.scala

```diff
@@ -52,13 +53,65 @@ class GpuGenerateExecSparkPlanMeta(
     }
   }
 
+  private def getExpandExecForStack(): GpuExec = {
+    val stackMeta = childExprs.filter(_.isInstanceOf[GpuStackMeta]).head
+    val numRows = extractLit(stackMeta.childExprs.head.convertToCpu()) match {
+      case Some(lit) => lit.value.asInstanceOf[Int]
+      case _ => throw new IllegalStateException("First parameter of stack should be a literal Int")
+    }
+    val numFields = Math.ceil((stackMeta.childExprs.length - 1.0) / numRows).toInt
+    val projections: Seq[Seq[Expression]] = for (row <- 0 until numRows) yield {
+      val childExprsGpu = childExprs.filterNot(_.isInstanceOf[GpuStackMeta])
```

Comment:
Is it possible there are other children for a GenerateExec ?

Reply:
Yes. There is only one generator in GenerateExec, but there may be other `requiredChildOutput`s.
```scala
override val childExprs: Seq[BaseExprMeta[_]] = {
  (Seq(gen.generator) ++ gen.requiredChildOutput).map(
    GpuOverrides.wrapExpr(_, conf, Some(this)))
}
```
I updated the `filterNot` to `tail`.